draft-ietf-taps-interface.md

```diff
@@ -653,23 +660,24 @@ when no transport-layer connection can be established to the Remote Endpoint,
 or when the application is prohibited from rendezvous by policy.
 
 When using some NAT traversal protocols, e.g., Interactive Connectivity
-Establishment (ICE) {{?RFC5245}}, it is
-expected that the Local Endpoint will be configured with some method of
-discovering NAT bindings, e.g., a Session Traversal Utilities for NAT (STUN) server.
-In this case, the
-Local Endpoint may resolve to a mixture of local and server reflexive
-addresses. The Resolve() action on the Preconnection can be used to
-discover these bindings:
+Establishment (ICE) {{?RFC5245}}, it is expected that the Local Endpoint will
+be configured with some method of discovering NAT bindings, e.g., a Session
+Traversal Utilities for NAT (STUN) server. In this case, the Local Endpoint
+may resolve to a mixture of local and server reflexive addresses. The
+Resolve() action on the Preconnection can be used to discover these bindings:
 
 ~~~
-PreconnectionBindings := Preconnection.Resolve()
+[]Preconnection := Preconnection.Resolve()
 ~~~
 
 The Resolve() call returns a list of Preconnection Objects, that represent the
```

Comment:
Just a nit: above, it's said that an action creates an array of objects, but here it's called a list. For consistency, it would be better to call it either a list or an array everywhere, I guess.